src/codex_plus/llm_execution_middleware.py

```
@@ -365,71 +455,698 @@ async def process_request(self, request, path: str):
+                async def stream_response_async():
+                    logger.info(
+                        "🔄 Cerebras transformer stream started",
+                        extra={"upstream": upstream_url},
+                    )
+                    logger.info(
+                        "🔍 Cerebras response status",
+                        extra={"status": response.status_code, "headers": dict(response.headers)},
+                    )
+
+                    try:
+                        log_dir = Path("/tmp/codex_plus/cerebras_responses")
```

Ruff (0.13.3)

Probable insecure usage of temporary file or directory: "/tmp/codex_plus/cerebras_responses"

(S108)